eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -1,280 +1,314 @@
+            "n_nsv_skipped_wrong_ref_alt": 0, "record_counter": 0, "n_total_clinvar_records": 0}
+
+
 def clinvar_to_evidence_strings(dir_out, allowed_clinical_significance=None, ignore_terms_file=None,
                                 adapt_terms_file=None, efo_mapping_file=None, snp_2_gene_file=None,
                                 variant_summary_file=None):
 
-    allowed_clinical_significance = allowed_clinical_significance.split(',') if allowed_clinical_significance else None
-
-    trait_2_efo, unavailable_efo_dict = load_efo_mapping(efo_mapping_file, ignore_terms_file, adapt_terms_file)
-
-    consequence_type_dict = consequence_type.process_consequence_type_file(snp_2_gene_file)
-    rcv_to_rs, rcv_to_nsv = clinvar_record.get_rcv_to_rsnsv_mapping(variant_summary_file)
-
-    if allowed_clinical_significance is None:
-        allowed_clinical_significance = ['unknown', 'untested', 'non-pathogenic', 'probable-non-pathogenic',
-                                       'probable-pathogenic', 'pathogenic', 'drug-response', 'drug response',
-                                       'histocompatibility', 'other', 'benign', 'protective', 'not provided',
-                                       'likely benign', 'confers sensitivity', 'uncertain significance',
-                                       'likely pathogenic', 'conflicting data from submitters', 'risk factor',
-                                         'association']
-    nsv_list = []
-    evidence_string_list = []
-    n_processed_clinvar_records = 0
-    n_pathogenic_no_rs = 0
-    n_multiple_evidence_strings = 0
-    n_multiple_allele_origin = 0
-    n_germline_somatic = 0
-    n_records_no_recognised_allele_origin = 0
-    no_variant_to_ensg_mapping = 0
-    n_more_than_one_efo_term = 0
-    n_same_ref_alt = 0
-    n_missed_strings_unmapped_traits = 0
-    n_nsvs = 0
-    n_valid_rs_and_nsv = 0
-    n_nsv_skipped_clin_sig = 0
-    n_nsv_skipped_wrong_ref_alt = 0
-    unmapped_traits = {}
-    n_unrecognised_allele_origin = {}
-    ensembl_gene_id_uris = set()
-    traits = set()
-    unrecognised_clin_sigs = set()
-    evidence_list = []  # To store Hellen Parkinson records of the form [RCV, rs, ClinVar trait, EFO url]
-    record_counter = 0
-    n_total_clinvar_records = 0
-    skip = 0
+    allowed_clinical_significance = allowed_clinical_significance.split(',') if allowed_clinical_significance else \
+        get_default_allowed_clincal_significance()
 
-    answer = urllib.request.urlopen('http://' + config.HOST + '/cellbase/webservices/rest/v3/hsapiens/feature/clinical/all?source=clinvar&skip=' + str(skip) + '&limit=' + str(config.BATCH_SIZE))
+    mappings = get_mappings(efo_mapping_file, ignore_terms_file, adapt_terms_file, snp_2_gene_file,
+                            variant_summary_file)
+
+    report = Report(unavailable_efo_dict=mappings.unavailable_efo_dict)
```

Comment:
No need to add the attribute name on the left side when it is the only optional argument.